chat-plugins/health-fitness.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * Heath & Fitness chat-plugin
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ * 
+ * This handles health and fitness daily challenges as well as a daily quote
+ * 
+ * @license MIT license
+ */
+'use strict';
+
+const FS = require('../fs');
+
+const HF_DATA_PATH = 'config/chat-plugins/health-and-fitness.json';
+const MAX_DATA_LENGTH = 500;
+
+class HealthFitness {
+	constructor() {
+		this.hfData = {
+			cardio: '',
+			gym: '',
+			quote: '',
+			__proto__: null,
```

Comment:
Unrelatedly: Why is this null-proto at all, if it's loaded from a JSON file?

Reply:
bump

Reply:
I've been busy with school, sorry. Uhh, I didn't realize that an object loaded from a JSON file wasn't susceptible to constructor crashes.

If this isn't the case, I'll update this later today.

Reply:
Objects loaded from JSON files are the same as any other object.

The issue is that you can't load null-prototype objects from JSON files. Which is why I instead use `hasOwnProperty` for them.